Profession Skill Points

marthammor opened the pull request:
Added locations for one-time use profession skill-ups. Still need quest ids for most to mark them as complete.
The points are only visible on the maps if you have the required Dragonflight professions learned.

marthammor committed:
Profession Skill Points

Added locations for one-time use profession skill-ups. Still need quest ids for most to mark them as complete.

MysticalOS commented:
Just adding https://www.wowhead.com/guide/professions/knowledge-point-treasure-locations-dragon-isles for reference :)

marthammor commented:
That's where I got most of the info. It's noted in the files. :)

marthammor committed:
Update

Updated notes for two inscription items due to a possible bug. Added correct(?) location for the jewelcrafting item, Painter's Pretty Jewel. Will confirm and get quest complete id later today for it and the recently fixed Lofty Malygite. I'll need someone else to get the quest ids for the jewelcrafting items Crystalline Overgrowth and Harmonic Crystal Harmonizer in Azure Span since I looted those before I knew how to see the quest ids.

kemayo commented:
This all looks good to have to me. It's going to be a bit of a pain getting characters with all the professions around to check for the questids, admittedly. 😄

marthammor commented:
Yeah, tell me about it. I do have chars with all the professions, it's just a matter of time to level them enough to grab the items. I did put the entry in for the quest ids, just set to nil atm. Just gotta plug them in when they're found.

kemayo commented:
Hm, I wonder if it's worth me adding something to the handler so we can indicate requiring a certain skill level -- it looks like most of these need a skill of 25 to collect. I double-checked the forgetful apprentice's tome I already had, and it doesn't, so that's not a universal requirement.

marthammor commented:
I don't know that it requires a specific level to loot (some comments on
wowhead says yes), but almost of the items themselves do say it requires
level 25 in the profession.

On Fri, Dec 2, 2022, 11:05 AM David Lynch ***@***.***> wrote:

> Hm, I wonder if it's worth me adding something to the handler so we can
> indicate requiring a certain skill level -- it looks like most of these
> need a skill of 25 to collect. I double-checked the forgetful apprentice's
> tome I already had, and it doesn't, so that's not a universal requirement.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/kemayo/wow-handynotes-dragonflight/pull/5#issuecomment-1335473702>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAG7PSJGQU5U67YUCHLWBBDWLIM23ANCNFSM6AAAAAASRSQZEQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

kemayo commented:
I double-checked some of the inscription ones with a character who doesn't have 25 skill yet, and that one apprentice tome was the only one I could pick up. I'll look into how complex adding a skill level check would be -- if it's easy, we can do it in a two-step way with `requires` for having the profession and then either `active` or `hide_before` for the skill level. (If it's excessively fiddly, the current way would be fine.)

marthammor commented:
Most of these have a container type item that the other usable item is in.
I wonder if those have a level limit that can be accessed somehow.

On Fri, Dec 2, 2022, 12:00 PM David Lynch ***@***.***> wrote:

> I double-checked some of the inscription ones with a character who doesn't
> have 25 skill yet, and that one apprentice tome was the only one I could
> pick up. I'll look into how complex adding a skill level check would be --
> if it's easy, we can do it in a two-step way with requires for having the
> profession and then either active or hide_before for the skill level. (If
> it's excessively fiddly, the current way would be fine.)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/kemayo/wow-handynotes-dragonflight/pull/5#issuecomment-1335538653>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAG7PSI677OZAKYBDI3YP2DWLITJFANCNFSM6AAAAAASRSQZEQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

kemayo commented:
Unfortunately, information about world objects is particularly hard to get out of the API. In this case it's not even an object with a "requires level 66" style warning, as some of the treasure-objects have -- they're just not showing as interactable at all (much like if you don't have the profession).

Good news, though, a profession level check is very tractable: https://github.com/kemayo/wow-handynotes-handler/commit/d0736aaae2256a79057495c609fe75c5dd70942d

kemayo commented:
Okay, I'm going to merge this as-is then do some rapid adjustments to add the relevant checks. (This seems easier than asking you to rebase the pull request / change the submodule within it.)

marthammor commented:
Awesome. I'll probably keep adding quest ids as I find them. I can do like
two more jewelcrafting ones today when I get home, but I already collected
the two in azure span before I found your other addon to get quest ids and
I don't have another jc to take out there, lol.

On Fri, Dec 2, 2022, 12:44 PM David Lynch ***@***.***> wrote:

> Merged #5 <https://github.com/kemayo/wow-handynotes-dragonflight/pull/5>
> into main.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/kemayo/wow-handynotes-dragonflight/pull/5#event-7942635833>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAG7PSJVVC4K2N4OR2EKGGTWLIYOJANCNFSM6AAAAAASRSQZEQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>

kemayo commented:
Okay, my adjustments are all pushed up. Feel free to either drop in the data in a raw form somewhere, or make pull requests and I'll merge them. 👍

kemayo commented:
I realized the existence of the one-time profession masters as well and added them in a55f17f, if you wanted to pick up relevant questids there as well...
